base/Base.jl

```diff
@@ -174,6 +174,7 @@ include("ctypes.jl")
 include("gcutils.jl")
 include("generator.jl")
 include("reflection.jl")
+include("reflection2.jl")
```

Comment:
@aviatesk proposed renaming to compiler_reflection.jl, which seems reasonable to me.

Reply:
Perhaps we should instead rename `reflection.jl` to `internals.jl`. I think that would be a better description of what it actually does: Provide accessors to various internal julia data structures.

Reply:
`internals.jl` sounds reasonable to me.

Reply:
Will rename accordingly.

Reply:
Something `compiler_internals.jl` would be better imo.

Reply:
It's not compiler internals though, it's internals of like method tables and other runtime datastructures.

Reply:
Right, this is more `runtime_internals.jl` or `cache_internals.jl` ?

Reply:
Ok, my point was mostly `xxx_internals.jl`.

Reply:
I'm ok with `runtime_internals.jl`